src/lib/php/Dao/CopyrightDao.php

```diff
@@ -159,6 +159,17 @@ public function undoDecision($tableName,$pfileId, $decisionId)
     $this->dbManager->execute(__METHOD__, array($decisionId, $pfileId));
   }
 
+  public function updateDecision($tableName,$pfileId, $decisionId, $content)
```

Comment:
I don't think this function is needed here. The function `saveDecision()` already performs the operation. Also, this function does not update the md5 of the updated text.

Reply:
Agreed, Removed the `updateDecision()` function.